src/components/course/data/constants.js

```diff
@@ -122,4 +122,5 @@ export const DISABLED_ENROLL_USER_MESSAGES = {
 /* eslint-enable max-len */
 
 export const DATE_FORMAT = 'MMM D, YYYY';
+export const DATETIME_FORMAT = 'MMM D, YYYY h:mm a';
```

Comment:
nit: i have no opinion one way or another, really, but the Figma wireframes show a comma between the date and time. Not sure if we wanted to have parity here?

Reply:
added it to stay keep in parity with the accepted documentation